src/common/intl/locales/fr.js

```diff
@@ -2025,8 +2024,8 @@ export default {
   // Original text: "Action"
   snapshotAction: 'Action',
 
-  // Original text: 'Quiesced snapshot'
-  snapshotQuiesce: undefined,
+  // Original text: "Quiesced snapshot"
+  snapshotQuiesce: 'Snapshot quiesce',
```

Comment:
I don't know how to translate *quiesce*.

But in the rest of XO, *snapshot* has been translated by *instantané*.